bitcoin-test/lib/Bitcoin/Crypto/SignatureSpec.hs

```diff
@@ -201,7 +204,7 @@ validImplMap =
 
 
 getImpl :: Maybe ValidImpl
-getImpl = implSig `Map.lookup` validImplMap
+getImpl = pure ImplCore
```

Comment:
Should we just completely get rid of the `ImplABC` stuff? Maybe that needs to be in a separate/follow-up PR.

Reply:
Yeah that's my plan for my next PR actually